src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/remove_convert.cpp

```diff
@@ -110,6 +124,15 @@ TEST_P(RemoveUselessConvertCPUTest, CompareWithRefs) {
     CheckNumberOfNodesWithType(compiledModel, "Convert", 0);
 }
 
+using RemoveUselessFP16ConvertCPUTest = RemoveUselessBF16ConvertCPUTest;
+TEST_P(RemoveUselessFP16ConvertCPUTest, CompareWithRefs) {
+    auto implType = get_default_imp_precision_type(ov::element::f16, configuration);
+    selectedType = makeSelectedTypeStr("ref", implType);
+    run();
+    CheckNumberOfNodesWithTypes(compiledModel, {"Convert", "Subgraph"}, 0);
+    CheckPluginRelatedResults(compiledModel, "StridedSlice");
```

Comment:
I will check it, the test case should not be run because it already add plugin_config on name, this skip_tests_config.cpp should skip it if platform doesn't support `fp16`

Reply:
from my test , the test case is skipped on platform without fp16 support

Reply:
already removed